src/demo/renderer.js

```diff
@@ -211,8 +220,78 @@ function predictionText(state, onComplete) {
   });
 }
 
+function initPondScreen(state) {
+  loadBackgroundImage().then(backgroundImg => {
+    renderBackgroundImage(backgroundImg);
+    drawPondScreen(state);
+    drawPondUiElements(state);
+  });
+}
+
 function drawPondScreen(state) {
-  console.log('pond!');
+  drawPondFish(state);
+}
+
+function drawPondFish(state) {
+  predictAllFish(state, fishWithConfidence => {
+    fishWithConfidence = _.sortBy(fishWithConfidence, ['confidence']);
+    const pondFish = fishWithConfidence.splice(0, 20);
+
+    pondFish.forEach(fishDatum => {
+      loadFishImages(fishDatum.fish).then(results => {
+        const randomX = randomInt(
+          FISH_CANVAS_WIDTH / 4,
+          state.canvas.width - FISH_CANVAS_WIDTH / 4
+        );
+        const randomY = randomInt(
+          FISH_CANVAS_HEIGHT / 4,
+          state.canvas.height - FISH_CANVAS_HEIGHT / 4
+        );
+        drawFish(fishDatum.fish, results, state.ctx, randomX, randomY);
+      });
+    });
+  });
+}
+
+function predictAllFish(state, onComplete) {
```

Comment:
We should be doing this in a model, rather than in the renderer, shouldn't we?

Reply:
good call, i'll get the model started and move this into it

Reply:
okay, i lied, i'm going to do this in a separate PR so this PR doesn't become enormous